### PR TITLE
[MIG][12.0] stock_production_lot_warranty 

### PR DESCRIPTION
**Pull request by murtuzasaleh**

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@osi-scampbell @max3903 Can you please review this PR.

File: stock_production_lot_warranty/models/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    warranty_exp_date = fields.Date(string='Warranty Expiration Date')
+
+    @api.onchange('product_id')
+    def _onchange_product_id(self):
+        self.warranty_exp_date = False
+        if self.product_id and \
+                self.product_id.product_tmpl_id.warranty_type and \
+                self.product_id.product_tmpl_id.warranty:
+            if self.product_id.product_tmpl_id.warranty_type == 'day':
```
Comment (rousseldenis) [inline stock_production_lot_warranty/models/stock_production_lot.py]: @murtuzasaleh You could maybe simplify writing:
```suggestion
            warranty_type = self.product_id.product_tmpl_id.warranty_type
            if  warranty_type == 'day':
```
and so on

Comment (murtuzasaleh) [reply stock_production_lot_warranty/models/stock_production_lot.py]: @rousseldenis Thanks, Done.

**Comment by rousseldenis** [comment]
@murtuzasaleh Could you update you PR first message to reference your dependence : https://github.com/OCA/rma/pull/142

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Could you update you PR first message to reference your dependence : #142

Done.

**Comment by rousseldenis** [comment]
@murtuzasaleh Can you rebase ?

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Can you rebase ?

Done.

**Review by osi-scampbell — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

**Pull request merged**